docs/UserGuide.md

```diff
@@ -147,36 +147,92 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st employee to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower` Edits the name of the 2nd person to be `Betsy Crower`.
 
-### Locating persons by name: `find`
+### Finding an employee by multiple attributes: `find`
 
 Finds the employees that satisfy the find requirements based on attributes.
 
 Format: `find [i/KEYWORD] [n/KEYWORD [OTHER KEYWORD]] [p/KEYWORD] [e/KEYWORD [OTHER KEYWORD]] [d/KEYWORD [OTHER KEYWORD]] [r/KEYWORD [OTHER KEYWORD]] [s/LOWERBOUND - UPPERBOUND]`
 
 * Find command must be used with **at least one prefix** and the keyword for the prefix **must not be empty**.
-* Refer to the prefix summary to understand which employee attribute corresponds with which prefix.
 * Find command for all attributes are **case-insensitive**. 
-* For finding by ID and Phone number, the command will find employees whose ID and Phone number contains the respective keywords.
-* For finding by Name, Email, Department and Role, the order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* For finding by Name, Department and Role, only **full words** will be matched e.g. `Han` will not match `Hans` and `Fina` will not match `Finance`.
-* For finding by Email, Email will be matched if it **contains the letter sequence** e.g. `alex` will match `alexyeoh@example.com`.
-* For finding by Name, Email, Department and Role, **matching at least one keyword** will be returned.
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang` e.g. `alex yeoh` will return `alextan@example.com`, `berniceyeoh@example.com`.
-* For finding by Salary, command will find employees whose salaries fall within the range of LOWERBOUND to UPPERBOUND inclusive of LOWERBOUND and UPPERBOUND.
-* The LOWERBOUND **cannot be larger than** the UPPERBOUND. 
-* There must be a **spacing before and after the dash**
-* LOWERBOUND and UPPERBOUND must be **non-negative integers** and **cannot be larger than 1 000 000 000**. 
+* Finds employees whose attributes satisfy all the requirements of the FindCommand
+
+Examples:
+* `find i/A00 n/alex p/9001` returns employees whose ID contains `A00`, whose name contains the word `alex` and whose phone number contains `9001`.
```

Comment:
Is it better to just use one "whose"? so
whose ID contains `A00`, name contains the word `alex` and phone number contains `9001`.

Reply:
Ok!